sound/MPlayDef.s

```diff
@@ -1,430 +1,416 @@
-	.equ	W00,	0x80		@ WAIT
```

Comment:
Not sure I'm a fan of removing these comments? Can't we just do `@;`?

Reply:
No we can't, because that'd still be a syntax error for those platforms.